### DISCUSSION TITLE
Auto-add to project - single repo scope

### DISCUSSION THREAD

**Discussion by ggnall** [community/community#44826] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Great to see the beta out for auto-add to project! One limitation that prevents us from using this in our projects is the ability to set rules across repos. 

So I'd like to define: if [label = x,y,z] in repos 1, 2, 3 -> auto-add to project. 

Since I can only scan one of several repos in the rule creation, we can't use this automation for the time being. And since it's simpler to add labels to an issue than add an issue to project(s), it would be really helpful to do so!

**Comment by 2percentsilk** [comment]
Thanks for the feedback @ggnall!

How many repos are you trying to pull from for a given project? We're looking at letting you create copies of auto-add (e.g. one workflow for each repo 👀

**Reply by ThomasKasene** [reply]
This sounds promising and very much needed! Is there an issue for this in the roadmap that we can follow?
For my case _at the moment_, I'm looking at ~10 repositories.

**Comment by ggnall** [comment]
Currently 3, but it could increase.

Cool, I would see myself adding duplicate workflows for each. It would be really cool to be able to use repos as a condition (E.g. in repos(...) AND has labels(...)).
> We're looking at letting you create copies of auto-add (e.g. one workflow for each repo 👀

**Comment by OneCyrus** [comment]
we are facing the same issue. currently we are linking about 15 repos to the same project and still increasing. would be great to have the limit much higher (100+)

**Reply by KevinEOSNation** [reply]
This is exactly what we need. The feature as-is now limits you to auto-adding 5 repos (unless you upgrade to Enterprise...but even that only gives you 20 repos)

**Comment by HazAT** [comment]
Same here, we still run our own GH Action to sync our project with multiple repos. It would be cool to get rid of this and GitHub natively adds this support for syncing multiple Repos with one Project.

**Reply by KevinEOSNation** [reply]
Hey...newbie here. How did you set that GH Action up?

**Comment by kathy-phet** [comment]
@2percentsilk - I saw that we can now "duplicate the workflow" and was temporarily excited but then disappointed ... we could work with this approach but the limit==4 is not working for us. We want to add issues from across all project repos (whole project) or across a subset of repos (e.g. 30 specific common code repos) automatically so need this limit to be much much higher. We have over 200 repos.

**Comment by rileybroughten** [comment]
👋 Hello all! Auto-add and auto-archive workflows are now available to all users, check out the announcement [here](https://github.blog/changelog/2023-03-09-github-issues-projects-march-9th-update/#robot-automatically-add-and-archive-project-items) as well as [documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/adding-items-automatically) for more details 🎉

**Reply by DFelten** [reply]
Is there a plan to remove this limitation? We work on significantly more repositories in parallel and would therefore need this workflow more often. It would be even more practical if multiple repositories could be selected in one workflow.

**Reply by Alfagun74** [reply]
the cap for free users on a single project is excessively low. i mean, nearly every project involves at least two repositories. restricting auto-add to one is distressing.

**Comment by brettcurtis** [comment]
For this to replace the GitHub action, the repo restrictions need to be removed. Teams will often prefix their repositories in a large organization. Something like `myteam-my-repo` and there could be several `myteam-*` repositories that align to a project. The dropdown for repositories needs to be replaced so we can simply add `myteam-*`.

Ideally, this would replace the action, not that the action is a lot of overhead really but it feels like a lot of noise in the actions all-workflow list.

**Comment by jsumners** [comment]
I've reached this issue because I want to write "in any org repo, for any issue that has title `A specific title`, add to project". As it is, I can only write that rule for individual repositories and the org I want this for has around 100 of them (https://github.com/fastify/fastify/issues/5116).

**Comment by Rnbsov** [comment]
oh my god, i wish so much it will be implemented 😢🥺
